R/correctionsataglance-data.R

```diff
@@ -128,9 +128,9 @@ formatDataList <- function(dataIn, type, ...){
   
   if(type == 'APPROVALS' && length(dataIn)>0){
     for (i in 1:length(dataIn$endTime)) {
-      if (as.Date(dataIn$endTime[i]) > "9000-12-31") {
+      if (as.Date(dataIn$endTime[i]) > "2050-12-31") {
         endT <- format(as.Date(dataIn$endTime[i]), format="%m-%d")
-        endT <- paste0("9000-",endT,"T00:00:00.000-06:00")
+        endT <- paste0("2050-",endT,"T00:00:00.000-06:00")
```

Comment:
if you look at the gsplot object, the rectangle positions are correct, but somehow it's showing up incorrectly?

If you just do a gsplot thing, it seems to work just fine:
```r
s <- c(as.POSIXct("1922-01-01 00:00:00"), as.POSIXct("2015-08-01 01:00:00"))
e <- c(as.POSIXct("2015-08-01 01:00:00"), as.POSIXct("9999-08-01 00:00:00"))
gs <- gsplot() %>%
    points(x = as.POSIXct(NA), y = NA, ylim=c(0,10), 
           xlim=c(as.POSIXct("2015-01-01"), as.POSIXct("2016-01-01"))) %>%
    rect(xleft = s, ybottom = 8, xright = e, ytop = 9, 
         col = c("green", "red"), border = NA)
gs
```